src/osd/PGBackend.cc

```diff
@@ -1041,7 +1053,7 @@ void PGBackend::be_compare_scrubmaps(
       }
       if (auth_object.digest_present && auth_object.omap_digest_present &&
 	  cct->_conf->osd_debug_scrub_chance_rewrite_digest &&
-	  (((unsigned)rand() % 100) >
+	  ((return_rand(errorstream) % 100) >
```

Comment:
...for example:
#include "include/random.h"
...
return ceph::util::generate_random_number(100); // returns [0-100], inclusive